### PR TITLE
add django fixtures for intial data

### LINKED ISSUE #162: Basic Data for User, Company, Garden, Beds
We need to seed standard data into the Database, so the api will work after migrating and starting the server.
https://docs.djangoproject.com/en/4.0/howto/initial-data/

### PR DESCRIPTION
**Pull request by jarkenau**
I added a way to initialize the Django backend with some default data e.g.  user, company. The data can be loaded with `python3 manage.py loaddata user company garden` or with the new command in the Makefile to start  the hole application. 

closes #162

As soon as the models are updated, the fixtures also need to be updated :exclamation:

### CODE REVIEW

**Comment by jarkenau** [comment]
@Anafabula The initial data is not working with the authentication, probably due to the password not getting hashed right?

File: django/restapi/fixtures/user.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+  },
+  {
+    "model": "auth.user",
+    "pk": 2,
+    "fields": {
+      "username": "dHaus",
+      "first_name": "David",
+      "last_name": "Haus",
+      "password": "1234",
```
Comment (Anafabula) [inline django/restapi/fixtures/user.json:19]:
```suggestion
      "password": "pbkdf2_sha256$320000$weVQAcFLezx0glHVvHyf5G$v8d6rFBAqdtwpS2ndWqOKCVqFrnbgnH86/LnZJueGTc=",
```

File: django/restapi/fixtures/user.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "model": "auth.user",
+    "pk": 1,
+    "fields": {
+      "username": "k.müller",
+      "first_name": "Kevin",
+      "last_name": "Müller",
+      "password": "1234"
```
Comment (Anafabula) [inline django/restapi/fixtures/user.json:9]:
```suggestion
      "password": "pbkdf2_sha256$320000$LuKlEyk1ZdbXx8ctkvF9c7$21bgLk3zi8fHBCZdTkAKQT8e0nIGe7A+gmS4zFaqzd4="
```

**Review by Anafabula — COMMENTED** [review]
> @Anafabula The initial data is not working with the authentication, probably due to the password not getting hashed right?

I assume, haven't tested it though

File: django/restapi/fixtures/user.json (hunk 1 above)
Comment (jarkenau) [reply django/restapi/fixtures/user.json:19]: Yeah works now, how did you get the hash, so we can include it as a comment?

Comment (Anafabula) [reply django/restapi/fixtures/user.json:19]: I added them via `/register` and looked in the database

**Pull request merged**